sdk/storage/test-resources.json

```
@@ -276,6 +272,14 @@
     "AZURE_CLIENT_SECRET": {
       "type": "string",
       "value": "[parameters('testApplicationSecret')]"
+    },
+    "DFS_ACCOUNT_NAME": {
+      "type": "string",
+      "value": "[variables('dataLakeAccountName')]"
+    },
+    "DFS_ACCOUNT_KEY": {
+      "type": "string",
+      "value": "[listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('dataLakeAccountName')), variables('mgmtApiVersion')).keys[0].value]"
```

`mgmtApiVersion` is not defined, updating it to `storageApiVersion`.
```suggestion
      "value": "[listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('dataLakeAccountName')), variables('storageApiVersion')).keys[0].value]"
```